Tests/TuistCoreTests/Graph/Nodes/TargetNodeTests.swift

```diff
@@ -49,14 +49,15 @@ final class TargetNodeTests: XCTestCase {
         XCTAssertNotEqual(c1, d)
     }
 
-    func test_encode() {
+    func test_encode() throws {
         // Given
         let library = LibraryNode.test()
         let framework = FrameworkNode.test()
         let cocoapods = CocoaPodsNode.test()
+        let xcframework = try XCFrameworkNode.test()
```

Comment:
Does this helper exist? is there a reason it's throwing?

Reply:
Thanks for spotting, i was running some trials, it got left over after. Will remove it.